schemas/iso19139.swe/src/main/plugin/iso19139.swe/update-fixed-info.xsl

```diff
@@ -900,6 +900,24 @@
     </gmd:role>
   </xsl:template>
 
+
+  <!-- if a gmd:ResourceMaintenance doesn't contain info (no codelist for MD_MaintenanceFrequencyCode and empty gmd:maintenanceNote, remove it
+	-->
+  <xsl:template match="gmd:resourceMaintenance">
+    <xsl:choose>
+      <!-- Remove gmd:resourceMaintenance if empty -->
+      <xsl:when test="gmd:MD_MaintenanceInformation/gmd:maintenanceAndUpdateFrequency/gmd:MD_MaintenanceFrequencyCode[@codeListValue='' or not(codeListValue)]
```

Comment:
The attribute misses the `@`, should be `not(@codeListValue)`@

Reply:
Fixed